<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connectionDialog</class>
 <widget class="QDialog" name="connectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки базы данных</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Пользователь</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edtPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edtUserName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblPassword">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;Пароль</string>
        </property>
        <property name="buddy">
         <cstring>edtPassword</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblUserName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;Имя</string>
        </property>
        <property name="buddy">
         <cstring>edtUserName</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="grpServer">
     <property name="title">
      <string>Сервер</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="6" column="1" colspan="2">
       <widget class="QCheckBox" name="chkCompressData">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Сжимать данные</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="btnDelConnectionInfo">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
          <strikeout>true</strikeout>
         </font>
        </property>
        <property name="toolTip">
         <string>Удалить конфигурацию подключения из списка</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../s11main.qrc">
          <normaloff>:/new/prefix1/icons/delete-icon.png</normaloff>:/new/prefix1/icons/delete-icon.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="edtConnectionName">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QSpinBox" name="edtServerPort">
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="edtDatabaseName"/>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="cmbDriverName">
        <item>
         <property name="text">
          <string>MySQL</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblDatabaseName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;База</string>
        </property>
        <property name="buddy">
         <cstring>edtDatabaseName</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblDriverName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;Тип</string>
        </property>
        <property name="buddy">
         <cstring>cmbDriverName</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblServerPort">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;Порт</string>
        </property>
        <property name="buddy">
         <cstring>edtServerPort</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblServerName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;Адрес </string>
        </property>
        <property name="buddy">
         <cstring>edtConnectionName</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblConnectionName">
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="edtServerName"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblLoggerDbName">
        <property name="text">
         <string>БД лог</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QLineEdit" name="edtLoggerDbName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="chkAutScheme">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Использовать индивидуальную схему авторизации</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cmbDriverName</tabstop>
  <tabstop>edtConnectionName</tabstop>
  <tabstop>btnDelConnectionInfo</tabstop>
  <tabstop>edtServerName</tabstop>
  <tabstop>edtServerPort</tabstop>
  <tabstop>edtDatabaseName</tabstop>
  <tabstop>chkCompressData</tabstop>
  <tabstop>edtUserName</tabstop>
  <tabstop>edtPassword</tabstop>
  <tabstop>chkAutScheme</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../s11main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>connectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>connectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
